frameworks/Dart/aqueduct/server.dart

```diff
@@ -0,0 +1,228 @@
+        .listen((req) async => new Response.ok("Hello, World!")
+          ..contentType = _stripped_text
+          ..headers["date"] = new DateTime.now());
+
+    router.route("/db").listen((req) async {
+      World result = await getRandomWorldObject();
+      return new Response.ok(result)
+        ..contentType = _stripped_json
+        ..headers["date"] = new DateTime.now();
+    });
+
+    router.route("/queries/[:queryCount]").listen((req) async {
+      int queryCount = _min_query_count;
+      if (req.path.variables.containsKey("queryCount")) {
+        queryCount = int
+            .parse(req.path.variables["queryCount"], onError: (_) => queryCount)
+            .clamp(_min_query_count, _max_query_count);
+      }
+      List<Future> resultFutures =
+          new List.generate(queryCount, (_) => getRandomWorldObject());
+      List results = await Future.wait(resultFutures);
+      return new Response.ok(results)
+        ..contentType = _stripped_json
+        ..headers["date"] = new DateTime.now();
+    });
+
+    router.route("/updates/[:queryCount]").listen((req) async {
+      int queryCount = _min_query_count;
+      if (req.path.variables.containsKey("queryCount")) {
+        queryCount = int
+            .parse(req.path.variables["queryCount"], onError: (_) => queryCount)
+            .clamp(_min_query_count, _max_query_count);
+      }
+      List<Future> resultFutures = new List.generate(
+          queryCount,
+          (_) => getRandomWorldObject().then((World world) async {
+                Query query = new Query<World>()
+                  ..where.id = whereEqualTo(world.id)
+                  ..values.randomNumber =
+                      (_random.nextInt(_world_table_size) + 1);
+                Future<World> result = query.updateOne();
+                return await result;
+              }));
+      List results = await Future.wait(resultFutures);
+      return new Response.ok(results)
+        ..contentType = _stripped_json
+        ..headers["date"] = new DateTime.now();
+    });
+
+    router.route("/fortunes").listen((req) async {
+      Query query = new Query<Fortune>();
+      List<Fortune> results = await query.fetch();
+      results.add(new Fortune()
+        ..id = 0
+        ..message = "Additional fortune added at request time.");
+      results.sort();
+
+      List resultsMapped = results
+          .map((Fortune fortune) =>
+              {'id': fortune.id, 'message': fortune.message})
+          .toList();
+
+      String renderedTemplate =
+          fortunesTemplate.renderString({'fortunes': resultsMapped});
+      return new Response.ok(renderedTemplate)
+        ..contentType = ContentType.HTML
+        ..headers["date"] = new DateTime.now();
+    });
+  }
+
+  Future<World> getRandomWorldObject() async {
+    int worldId = _random.nextInt(_world_table_size) + 1;
+    Query query = new Query<World>()..where.id = worldId;
+    Future<World> result = query.fetchOne();
```

Comment:
Here, you can simply `return query.fetchOne()` instead of storing in a var and then returning that var